pkg/hook/hook.go

```diff
@@ -55,8 +57,19 @@ func (h *Hook) Dump() ([]byte, error) {
 	return yaml.Marshal(h)
 }
 
+func (h *Hook) joinURLParams() string {
```

Comment:
Once you switch to using [url.Values](https://godoc.org/net/url#Values), you can replace this method with `h.Params.Encode()`